src/Fixer/Alias/ModernizeStrposFixer.php

```diff
@@ -163,6 +166,8 @@ private function fixCall(Tokens $tokens, int $functionIndex, array $operatorIndi
 
     /**
      * @param -1|1 $direction
+     *
+     * @return array{operator_index: int, operand_index: int}
```

Comment:
Not sure why this is not reported:
```suggestion
     * @return null|array{operator_index: int, operand_index: int}
```

Reply:
I agree, great catch, not sure why SCA didn't trip